_windows/git.xml

```diff
@@ -0,0 +1,6 @@
+<application>
```

Comment:
There seems to be a lot of .xml files being added by your IDE, can you add these into the .gitignore file so they aren't committed?

Reply:
fixed